src/sass/form.scss

```diff
@@ -0,0 +1,29 @@
+.iti-flag {
+    background-image: url("/img/flags.png");
+}
+
+@media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
```

Comment:
try to not use browser prefix. Look at SASS output then

Reply:
Noted